server/routes/api/auth/strategy/googleAuth.js

```diff
@@ -1,10 +1,22 @@
 import {
   Router
 } from 'express';
-import GoogleAuth from '../../../../helpers/auth/googleAuthenticate';
+import passport from 'passport';
+
+import socialMediaControllerCallback from
+'../../../../helpers/socialMediaControllerCallback';
+
 
 const router = Router();
 
-router.get('/auth/google', GoogleAuth.googleRoute());
+router.get('/auth/google', passport.authenticate('google', {
+  scope: ['profile', 'email']
+}));
+router.get(
+  '/auth/google/callback',
+  passport.authenticate('google', {
+    failureRedirect: '/',
+    session: false
+  }), (req, res) => socialMediaControllerCallback (req, res));
```

Comment:
Hi @JohnMadakin. I don't think you need to do `(req, res) => socialMediaControllerCallback(req, res)`. You could have just done `socialMediaControllerCallback(req, res)`. Am open to correction though.

Reply:
if you do `socialMediaControllerCallback(req, res)`, you are executing the callback while passing 2 undefined objects. it will through res undefined error.

Reply:
Oh okay